src/game/core/explore.js

```diff
@@ -43,7 +43,7 @@ export function exploreUntilDead (dao, player, gameMap, char) {
           fight,
           identity,
         ))
-        .catch(() => console.log('Invalid combat token, refusing to save'))
+        .catch(e => { console.log(e); console.log('Invalid combat token, refusing to save') })
```

Comment:
Maybe `.catch(console.log.bind(console, 'Invalid combat token, refusing to save'))` ?